### PR TITLE
Add note about flexbox to readme

### PR DESCRIPTION
**Pull request by joelochlann**
Following on from the excellent work around browser support (#541), I wanted to make it clear that we **can** now use flexbox - with a few IE11-related caveats.

This is also an argument for making IE11 the priority to be dropped when we next do a review of browser support

@davidfurey @Ap0c @JustinPinner @rupertbates @jranks123 @Mullefa

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -464,8 +464,16 @@ If you are firing a conversion event for a specific test, be sure that the `part
 |   `Prod`  |        `true`       |   `Dev`  | `Dev`    |       `Prod`      | `UAT`   |
 |   `Prod`  | `false` (real user) |  `Prod`  | `Prod`   |       `Prod`      | `Prod`  |
 
-## 10 CSS guidelines
+## 10 CSS
```
Comment (svillafe) [inline docs/development.md]: By removing "guidelines", the link from the top of this file breaks and the link form the project's Readme file breaks too. Please update those.

Comment (joelochlann) [reply docs/development.md]: ah sorry!

**Comment by Ap0c** [comment]
@joelochlann Safari 8 doesn't have support for flexbox, it only provides it prefixed. Currently I don't think our autoprefixer config is producing prefixes for Safari 8; I suspect that it's not in their default browser list because the audience share is low globally. If we want to start using flexbox more widely we'll have to update our config to provide the `-webkit` prefixes.

**Comment by Ap0c** [comment]
It [looks like](https://github.com/postcss/autoprefixer#browsers) using a `.browserslistrc` to specify what browsers we support will work for both Babel and Autoprefixer, so maybe we could consider that?

File: docs/development.md (hunk 1 above)
Comment (joelochlann) [reply docs/development.md]: sorted

**Comment by svillafe** [comment]
@Ap0c a PR with the correct configuration of babel and autoprefixier is coming 😄

**Comment by Ap0c** [comment]
A good way to check if flexbox is working in a particular browser is to go to the one-off thank you page on a fairly large screen. If it's working the page should be full height, if not there'll be a big gap below the footer. This happens because we're [using flexbox](https://github.com/guardian/support-frontend/blob/master/assets/stylesheets/gu-sass/layout.scss) to push the footer to the bottom of the viewport on short pages.

Sorry @joelochlann, I'll stop commenting on your PR now =).

Also, @svillafe 👍

**Comment by joelochlann** [comment]
Thanks @Ap0c, using `.browserslistrc` is a great call

**Comment by svillafe** [comment]
@Ap0c great tip about that thank you page!!

File: docs/development.md (hunk 1 above)
Comment (joelochlann) [reply docs/development.md]: ... and the readme too .... sorry!

**Pull request merged**